packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -583,27 +626,76 @@ class MemoryController extends DisposableController
         ?.getHeapSnapshotGraph(serviceManager?.isolateManager?.selectedIsolate);
   }
 
+  final _monitorAllocationsUpdated = ValueNotifier<int>(0);
+
+  /// Last column sorted and sort direction in allocation monitoring. As table
+  /// is reconstructed e.g., reset, etc. remembers user's sorting preference.
+  ColumnData<ClassHeapDetailStats> sortedMonitorColumn;
+  SortDirection sortedMonitorDirection;
+
+  ValueListenable<int> get monitorAllocationsChanged =>
+      _monitorAllocationsUpdated;
+
+  DateTime monitorTimestamp;
+
+  var _monitorAllocations = <ClassHeapDetailStats>[];
+
+  List<ClassHeapDetailStats> get monitorAllocations => _monitorAllocations;
+
+  set monitorAllocations(List<ClassHeapDetailStats> allocations) {
+    _monitorAllocations = allocations;
+    // Clearing allocations reset ValueNotifier to zero.
+    allocations.isNotEmpty
+        ? _monitorAllocationsUpdated.value++
+        : _monitorAllocationsUpdated.value = 0;
+  }
+
   Future<List<ClassHeapDetailStats>> resetAllocationProfile() =>
       getAllocationProfile(reset: true);
 
-  // 'reset': true to reset the object allocation accumulators
+  /// 'reset': true to reset the object allocation accumulators
   Future<List<ClassHeapDetailStats>> getAllocationProfile({
-    bool reset = false,
+    reset = false,
```

Comment:
add back `bool`. I don't think Dart infers that reset has to have type bool in this case but I could be wrong.

Reply:
Done.